shared/src/business/utilities/htmlGenerator/fonts.scss

```diff
@@ -135,5 +135,5 @@
 /* stylelint-enable */
 
 .nimbus {
-  font-family: 'nimbus_roman';
+  font-family: 'nimbus_roman', serif;
```

Comment:
This was changed as part of [another PR](https://github.com/ustaxcourt/ef-cms/pull/1972), but the linter was not run. This corrects a lint error.